public/serviceWorker.js

```diff
@@ -89,15 +89,29 @@ firebase.initializeApp(firebaseConfig);
 const messaging = firebase.messaging();
 messaging.usePublicVapidKey('BESZjJEHTRUVz5_8NW-jjOToWiSJFZHDzK9AYZP6No8cqGHkP7UQ_1XnEPqShuQtGj8lvtjBlkfoV86m_PadW30')
 messaging.onBackgroundMessage(function(payload) {
-    const notificationTitle = payload.notification.title;
-    const notificationOptions = {
-        body: payload.notification.body,
-        icon: '/logo192.png',
-        data: {
-            url: payload.data ? payload.data.onClick : "https://sky.coflnet.com"
-        }, //the url which we gonna use later
+    const request = indexedDB.open('keyval-store', 1);
+    var db;
+    if (payload.data.type != "auction") {
+        console.log("unkown notification type" + payload.data.type);
+        return;
+    }
+
+    var auction = JSON.parse(payload.data.auction);
+
+
+    var key = `auctionDetails\"${auction.uuid}\"`;
+    var data = {
+        expireTimeStamp: (new Date()).getTime() + 60 * 1000,
```

Comment:
remove unnecessary brackets

Reply:
wich brackets are unnecessary here?

Reply:
Its the same as here: https://github.com/matthias-luger/hypixel-react/blob/master/src/utils/CacheUtils.tsx#L24